src/main/webapp/features/screens/game/view.html

```diff
@@ -1,48 +1,3 @@
-<div class="row">
+<ct-game-room ng-if="game.isNew()" game="game"></ct-game-room>
```

Comment:
I would rather split it on 2 different views resolved by ui-router on different pathes, for example 'game-details/123' and 'game-play/123'